src/docs/_sidebars/arcCommanderManualSidebar.md

```diff
@@ -1,17 +1,32 @@
 ---
```

Comment:
The new structure is so much better! Great job! 
Just the wording could be improved... see below.

Reply:
Thanks and agreed. This is just a first iteration and more to come / cut down.